Default.sublime-commands

```diff
@@ -118,6 +118,16 @@
         "args": { "in_cached_mode": true }
     },
     {
+        "caption": "git: diff current file",
+        "command": "gs_diff",
+        "args": { "in_current_file": true }
+    },
+    {
+        "caption": "git: diff cached current file",
```

Comment:
Change to `git: diff current file (cached)` to be consistent with other palette commands.